plugin/www/AppboyPlugin.js

```diff
@@ -93,6 +93,14 @@ AppboyPlugin.prototype.submitFeedback = function (email, feedback, isBug) {
 
 // Appboy user methods
 /**
+ * Sets the user Attribution Data. We are using this to connect our integration with Adjust
```

Comment:
Yeah, I wrote it too fast, and I even forgot to write the comments for the params. I will get to that, the comments on the other method are so helpful